### DISCUSSION TITLE
Perform action after x-for has finished

### DISCUSSION THREAD

**Discussion by TomS-** [alpinejs/alpine#1206] [4. General]
Hello, 

I am implementing a pre-processor type thing to allow Alpine to work in Webflow. It's going pretty great but I want to repaint Webflow after x-for has finished so things like sliders, parallax... etc. still work.

Is there any way of watching for an x-for to finish?

EDIT:
I imagine this would be the same thing for Isotope.js

**Comment by KevinBatdorf** [comment]
Have you tried using $nextTick()?
